client/src/components/common/footer/style.css

```diff
@@ -0,0 +1,53 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
+.footer {
+  background-color: #eff6f5;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  font-size: 16px;
+  text-align: left;
+  padding: 4em 8vw 2em 6vw;
+}
+
+.footer__p1 {
+  font-size: 1em;
+  font-family: "Raleway", sans-serif;
```

Comment:
Instead of repeating the font family everywhere, you can add it once in the `body` or `*` tags.
```css
body {
  font-family: myFont;
}

/* or */

* {
  font-family: myFont;
}
```